src/lib/config.ts

```diff
@@ -172,7 +173,8 @@ interface BasicConfig {
 	 *
 	 * @see https://marked.js.org/using_pro#extensions
 	 */
-	marked_extensions: marked.MarkedExtension[];
+	// marked_extensions: (marked.TokenizerExtension | marked.RendererExtension | (marked.TokenizerExtension & marked.RendererExtension))[];
+	marked_extensions: (marked.TokenizerExtension | marked.RendererExtension | (marked.TokenizerExtension & marked.RendererExtension))[];
```

Comment:
Isn't this the same as

```suggestion
	marked_extensions: marked.MarkedExtension[];
```

?

Reply:
Apparently no, with `marked.MarkedExtension` I have the following error:
```console
src/lib/get-marked-with-highlighter.ts:14:15 - error TS2322: Type 'MarkedExtension[]' is not assignable to type '(TokenizerExtension | RendererExtension | (TokenizerExtension & RendererExtension))[]'.
  Type 'MarkedExtension' is not assignable to type 'TokenizerExtension | RendererExtension | (TokenizerExtension & RendererExtension)'.
    Type 'MarkedExtension' is not assignable to type 'TokenizerExtension & RendererExtension'.
      Type 'MarkedExtension' is missing the following properties from type 'TokenizerExtension': name, level

14  marked.use({ extensions });

```

Reply:
Ah ok, I just had a look at the types, so there's two options... either we change it to

```suggestion
	marked_extensions: marked.MarkedExtension['extensions'][];
```

or we leave it and change the default config to

```ts
{
  // ...
  marked_extensions: [{ extensions: [markedEmoji] }],
}
```

Reply:
I like the second idea, I just commited it :+1: